src/graphql/query/basicLoanData.graphql

```diff
@@ -30,4 +31,15 @@ query basicLoanData(
 			}
 		}
 	}
+	shop (basketId: $basketId) {
```

Comment:
Could you please use the existing separate basketItems query rather than add this here? You should be able to make this separate query as part of your query chain or independently to get this information when you need it.

I don't think we should tightly couple generic loan data fetches with shop basket information.